src-frontend/app/authenticators/token.js

```diff
@@ -0,0 +1,37 @@
+import Ember from 'ember';
+import Base from 'simple-auth/authenticators/base';
+import ENV from 'src-frontend/config/environment';
+
+export default Base.extend({
+    restore(data) {
+        return new Ember.RSVP.Promise(function(resolve, reject) {
+            if (!Ember.isEmpty(data.token)) {
+                resolve(data);
+            } else {
+                reject();
+            }
+        });
+    },
+
+    authenticate(credentials) {
+        return new Ember.RSVP.Promise(function(resolve, reject) {
+            Ember.$.ajax({
+                method: 'POST',
+                url: ENV.APP.API_HOST + '/auth/login',
+                data: credentials
+            }).then(function(response) {
+                Ember.run(function() {
+                    resolve({ token: response.token });
+                });
+            }, function(xhr, status, error) {
+                Ember.run(function() {
+                    reject(xhr);
```

Comment:
This causes the promise to go unfulfilled, which tells `ember-simple-auth` that authentication was unsuccessful. We don't currently do anything in this case, but we already have a TODO for showing a banner or something.